mongodb-advanced/index.js

```diff
@@ -40,7 +40,7 @@ function addRoutes(app) {
       if(err) {
         return next(err);
       }
-      res.send(result.result);
+      res.status(201).end();
```

Comment:
when 201 is used, it should have the location header set as described here: https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html

For the purposes of this example, I think `res.sendStatus(200);` would be appropriate.

In the case of `delete` we want to use what you have `res.status(204).end();` because `sendStatus` sends a body when used with 204, which it should not do, which is why we don't use `sendStatus` in that instance.